Base/res/html/misc/worker.js

```diff
@@ -0,0 +1,9 @@
+onmessage = (evt) => { 
+	postMessage(JSON.stringify(evt.data));
+	console.log("In Worker - Posted message");
+};
```

Comment:
No tabs for indentation please - run prettier over this file.

Reply:
also extra space here: `=> { ` <-----
:P